<ui version="4.0" >
 <class>FormOduncVer</class>
 <widget class="QWidget" name="FormOduncVer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Kitap Ödünç Ver</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="ikon.qrc" >:/ikonlar/share.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Üye Ara:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="ara_uyeler" />
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Kitap Ara:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QLineEdit" name="ara_kitaplar" />
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tVuyeler" >
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tVKitaplar" >
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Seçilen Üye:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="lEUye" />
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Seçilen Kitap:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QLineEdit" name="lEKitap" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Verme Tarihi:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QDateEdit" name="dateVerme" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Alınacak Tarih:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QDateEdit" name="dateAlma" >
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QPushButton" name="pBver" >
     <property name="text" >
      <string>Ödünç &amp;Ver</string>
     </property>
     <property name="icon" >
      <iconset resource="ikon.qrc" >:/ikonlar/onayla.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ikon.qrc" />
 </resources>
 <connections/>
</ui>
